tck/src/main/java/org/eclipse/microprofile/lra/tck/TckLRATypeTests.java

```diff
@@ -286,10 +311,14 @@ private void resourceRequest(String path, boolean startLRA, int expectedStatus,
             if (lra != null) {
                 lraClient.closeLRA(lra);
             }
+            
+            return methodLRA;
         } catch (Throwable e) {
             LOGGER.warning(e.getMessage());
         } finally {
             response.close();
         }
+        
+        return null;
```

Comment:
This might be wrong but I am wondering, should the "return null;" be moved under LOGGER.warning(e.getMessage()); as I think that is the only branch when you would return null?

Reply:
can do, thanks

Reply:
Thanks!